src/content/changes/highlights/2021-06-04.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2021-06-04T14:00:00
+title: Highlights for the week ending Jun 06, 2021
+changes_categories:
+- Highlights
+owner:
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+We fixed issues related to preparing for providing Management API access. These include correcting the chart name when installing an app and adding a missing import in the ingress installation step. We also upgraded the `ws` dependency to address a security vulnerability.
+
+## Documentation
+
+We wrote an article on [Organizations](https://docs.giantswarm.io/general/organizations/), as defined in the Management API. These replace the organizations as used with the REST API. Please read the details of this transition carefully to understand the ramifications for you and your end users.
+
+These weekly announcements now also appear in the Changes and Releases section, as their own category named [Highlights](https://docs.giantswarm.io/changes/highlights/).
```

Comment:
This appears 2nd time now, but I think it's worth repeating, maybe up to 4x (?)

Reply:
I think after this one, we are fine. This is not critical info, so if some people miss this, that's fine.

Reply:
ok!